src/hooks/useLocalPaginator.ts

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+
+export interface PageInfo {
+  endCursor: string;
+  hasNextPage: boolean;
+  hasPreviousPage: boolean;
+  startCursor: string;
+}
+
+export interface PaginationState {
+  after?: string;
+  before?: string;
+  first?: number;
+  last?: number;
+}
```

Comment:
I believe I didn't get to review this piece of code but `PageInfo` and `PagiationState` types are defined somewhere in the app and I don't think they should be redefined here

Reply:
will be changed in other pr